### DISCUSSION TITLE
How to disable enter key for the button of survey-html-form plugin?

### DISCUSSION THREAD

**Discussion by zx23237** [jspsych/jsPsych#1402] [Q&A]
Hi everyone,

When I use the survey-html-form plugin to fill a form, I usually press the enter key after I made an input. But I found that when I press the enter key, it will go to the next page instead of next input box. Does anyone have the idea about disable the enter key from going to the next page in the plugin? Thanks!

**Comment by jodeleeuw** [comment]
Hi @zx23237,

This [StackOverflow answer](https://stackoverflow.com/a/51507806/3726673) looks like a promising approach that might not require any plugin modifications.
